doc/api/single-executable-applications.md

```diff
@@ -153,7 +153,8 @@ platforms:
 
 * Windows
 * macOS
-* Linux (AMD64 only)
+* Linux (all distributions except Alpine and all architectures except s390x and
+  ppc64)
```

Comment:
Maybe we should be careful and not pretend "all but". We certainly do not test on all possible distributions and architectures that exist in the wild.

Reply:
I've updated the doc to clarify that this is limited by the set of platforms that Node.js supports. Does this look clearer now?

Reply:
Yes, thank you.